### DISCUSSION TITLE
Menu Bar Blur not Shown on Screenshots

### DISCUSSION THREAD

**Discussion by es183923** [FelixKratz/SketchyBar#66] [Q&A] [answered]
With the following config, screenshots show the original menu bar. 
```bash

sketchybar -m batch \
    --bar \
        height=25 \
        blur_radius=100 \
        topmost=on \
        bar_color="0x11000000"
```
![Screen Shot 2021-09-28 at 10 11 07 PM](https://user-images.githubusercontent.com/72744903/135191439-4b511714-1d08-4aa8-9825-8b4a96a6eb65.png)

By setting `barcolor` to `0xFF000000`, the original menu is completely covered.
Is it possible to prevent the menu bar from showing when sketchybar is still transparent by somehow adding the `blur_radius` factor into it?

**Comment by FelixKratz** [comment, marked as answer]
You can activate auto hide for the menu bar in the system prefs and disable topmost. The blur will not be present on screenshots either, but at least there is no bar beneath. I don‘t know why the blur is not recorded, I use private and undocumented functions to create the blur.

**Comment by es183923** [comment]
Could you add a disclaimer in the README for this so that others will know?
